charms/kserve-controller/requirements-integration.in

```diff
@@ -1,7 +1,6 @@
 aiohttp
 jinja2
-# Pinning to <3.0 due to compatibility with the 2.9 controller version and 3.0 not being maintained anymore
-juju<3.0
+juju==2.9.44
```

Comment:
Looks like this was inherited from a cherry-picked commit, I think it is preferable to keep the pin `juju<3.0`